frontend/src/components/admin/Admin.js

```diff
@@ -57,6 +59,9 @@ function Admin() {
           <SideNavLink renderIcon={ChartBubble} href="#program">
             <FormattedMessage id="sidenav.label.admin.program" />
           </SideNavLink>
+          <SideNavLink renderIcon={User} href="#userManagement">
+            <FormattedMessage id="unifiedSystemUser.browser.title" />
```

Comment:
Hello @adityadeshlahre , some merge conflicts to fix

Reply:
i will fix it once PR is 100% ready :slightly_smiling_face: